pkg/apis/experimental/install/install.go

```diff
@@ -51,14 +51,16 @@ func init() {
 		GroupVersion: groupVersion,
```

Comment:
because we ask people to specify the versions in the order of the most preferred first, 

https://github.com/kubernetes/kubernetes/blob/master/pkg/api/registered/registered.go#L42